locations/spiders/asics_eu.py

```diff
@@ -0,0 +1,20 @@
+import scrapy
+
+from locations.dict_parser import DictParser
+
+
+class AsicsEuSpider(scrapy.Spider):
+
+    name = "asics_eu"
+    item_attributes = {"brand": "asics", "brand_wikidata": "Q327247"}
+    allowed_domains = ["www.asics.com"]
+    start_urls = ["https://cdn.crobox.io/content/ujf067/stores.json"]
+
+    def parse(self, response):
+        for store in response.json():
+            if store["storetype"] in ["factory-outlet", "retail-store"]:
+                item = DictParser.parse(store)
+                item["street_address"] = item.pop("addr_full")
+                item["ref"] = store["name"]
+                item["brand"] = f'ascis-{store["storetype"]}'
```

Comment:
```suggestion
                item["brand"] = f'asics-{store["storetype"]}'
```

This should be a human-readable name. Is the brand known as `asics-factory-outlet`? It probably should be "Asics Factory Outlet".